ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/activity-redesigned.component.ts

```diff
@@ -156,4 +171,8 @@ export class ActivityRedesignedComponent extends ActivityComponent implements On
         injector: Injector) {
         super(windowRef, renderer, submitService, analytics, document, injector);
     }
+
+    public isAgree(): boolean {
+        return this.model.activityCode === 'CONSENT';
```

Comment:
@yufengwng, it looks like we must give the TestBoson consent activity a unique activityCode not to affect other studies.

Reply:
Here's a wild idea:

* Make `useAgreeConsent: boolean` an `@Input` for `ActivityRedesignedComponent`
* Make `isAgreeConsent()` method check for `useAgreeConsent` and `model.formType === 'CONSENT'`
* Add optional `useAgreeConsent` to toolkit config
* Change `ActivityPageRedesignedComponent` in toolkit to read `useAgreeConsent` from config and pass flag accordingly to `ddp-activity-redesigned`

Will that work?